### PR TITLE
Fix/build output layout

### PR DESCRIPTION
**Pull request by chmelevskij**
* fix `ol.css` for nwjs build
* fix race condition vite global vars
*

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
For reference

image

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**
